Formula/mujs.rb

```diff
@@ -18,12 +19,14 @@ class Mujs < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "33d9cd1bca8a34431d33b59b5ad5bc3d245fc288646b39e586577fcf43df7e33"
   end
 
+  depends_on "pkg-config" => :test
+
   on_linux do
     depends_on "readline"
   end
 
   def install
-    system "make", "release"
+    system "make", "prefix=#{prefix}", "release"
```

Comment:
```suggestion
    system "make", "prefix=#{opt_prefix}", "release"
```
Let's use `opt_prefix` here to use `opt_prefix` references in the `.pc` file. The new `test` will need adjusting.

Reply:
updated, thanks